Rc 10.5.3

zaibon opened the pull request.

xmonader commented on Jumpscale/clients/nbh_vertex/NBHClient.py:
```diff
@@ -0,0 +1,424 @@
+import requests
+import json
```
can we move to using j.data.serializers.json through the client?

xmonader commented on Jumpscale/clients/nbh_vertex/NBHClient.py:
```diff
@@ -0,0 +1,424 @@
+import requests
```
i think jumpscale wraps it using j.clients.http

serboctor replied:
I don't see anything there regarding sessions

xmonader commented on Jumpscale/clients/nbh_vertex/NBHClient.py:
```diff
@@ -0,0 +1,424 @@
+    if isinstance(data, str) or isinstance(data, int):
+        error_string = get_error_string(str(data))
+        if error_string:
+            raise Exception(error_string)
```
let's use j.exceptions.* types

despiegk committed:
doc improvement

despiegk committed:
Merge branch 'development' of github.com:threefoldtech/jumpscaleX_core into development

# Conflicts:
#	docs/3sdk/3sdk_install.md

m-motawea committed:
show actor path on gedis actor_add error (#795)

* show actor path on gedis actor_add error

* add msg to exceptions in GedisCmds

waleedhammam committed:
update windows installation docs

waleedhammam committed:
add links for docker and git install in windows

waleedhammam committed:
black formatting

waleedhammam committed:
update docker configuration for windows

grimpy committed:
Don't show shell path when opening kosmos

Signed-off-by: Jo De Boeck <deboeck.jo@gmail.com>

m-motawea committed:
fix j not defined in 3sdk

m-motawea committed:
add allow_empty flag to ask methods (#800)

waleedhammam committed:
add timepicker component for chatflows (#801)

waleedhammam committed:
add min duration in reservation to be 1 hour

waleedhammam committed:
update datetime picker message

AhmedHanafy725 committed:
Update zeroci yaml file

AhmedHanafy725 committed:
Fix zeroci yaml file

m-motawea committed:
fix int_ask allow_empty flag

rkhamis committed:
Merge pull request #804 from threefoldtech/development_update_ci_config

Update zeroci config

waleedhammam committed:
edit the usage of installtools to align with windows

windows doesn't have symlinks so it fails

 will try import from 3sdk first if not will use the old import

waleedhammam committed:
Merge branch 'rc_10.5.3' into development_windowshotfix

rkhamis committed:
Merge pull request #806 from threefoldtech/development_windowshotfix

Development windowshotfix
